src/components/tooltip/markers/NetworkMarkerTooltipContents.js

```diff
@@ -234,12 +243,10 @@ export class NetworkMarkerTooltipContents extends React.PureComponent<Props> {
           <TooltipDetail label="Priority">
             {_getHumanReadablePriority(payload.pri)}
           </TooltipDetail>
-          <TooltipDetail label="MIME type">
-            {guessMimeTypeFromNetworkMarker(payload)}
-          </TooltipDetail>
+          <TooltipDetail label="MIME type">{mimeType}</TooltipDetail>
```

Comment:
I feel pretty strongly that we should have a color swatch here, as the color is determined by the apparent MIME type.

Reply:
In fact, I'm wondering if you just forgot, as you renamed the `category-swatch` class, and I don't see where it's used in any new places.

Reply:
ahah yeah, exactly :D